$OpenBSD: patch-pkcs11_secret-store_gkm-secret-binary_c,v 1.1 2011/09/12 07:28:53 jasper Exp $

warning: passing argument 4 of 'buffer_get_time' from incompatible pointer type
(https://bugzilla.gnome.org/show_bug.cgi?id=652565)

--- pkcs11/secret-store/gkm-secret-binary.c.orig	Tue May 24 10:06:56 2011
+++ pkcs11/secret-store/gkm-secret-binary.c	Sun Jul  3 17:45:29 2011
@@ -71,8 +71,8 @@ typedef struct {
 	char *display_name;
 	const guchar *ptr_secret;
 	gsize n_secret;
-	time_t ctime;
-	time_t mtime;
+	long int ctime;
+	long int mtime;
 	GHashTable *attributes;
 	GList *acl;
 } ItemInfo;
@@ -849,7 +849,7 @@ gkm_secret_binary_read (GkmSecretCollection *collectio
 	guchar major, minor, crypto, hash;
 	guint32 flags;
 	guint32 lock_timeout;
-	time_t mtime, ctime;
+	long int mtime, ctime;
 	char *display_name;
 	guint32 tmp;
 	guint32 num_items;
